ADMVEDU/ViewControllers/FilterViewController.swift

```diff
@@ -10,13 +10,13 @@ import UIKit
 class FilterViewController: UIViewController {
 
```

Comment:
Please mark all methods with "private" where needed. Do it for all other files too